rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -30,12 +33,16 @@
     function render() {
       var $rapidBlockContent = $element.find(rapidBlockContentSel);
       $rapidBlockContent.html(toggleTemplate(state));
-      renderD3(state);
+      renderD3();
 
       $rapidBlockContent.find(problemStatusBtnSel).click(function() {
         $.post(toggleStatusUrl).then(
           function(newState) {
-            state = _.assign({}, state, newState);
+            // if the button to toggle this is visible there should only be one chart, so
+            // selectedRuns just replaces the existing one
```

Comment:
Confused about this comment. The 3 lines below look like they just reset the selection of runs, which makes sense to me. Not sure what is meant be the toggle button being visible and selectedRuns replacing the existing one

Reply:
I meant that in this situation `selectedRuns` should only have one item in it, so we can just replace it entirely and not worry about editing only the chart we care about

Reply:
I guess I'm nitpicking, but what about just "Selected runs should be reset when the open status is changed" for the comment text? The button visibility bit seems unnecessary as this is the click handler for that very button, and I'm not sure we need to know that the `selectedRuns` array has just one item at this point

Reply:
Sounds good